bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -45,6 +46,40 @@ void bugsnag_remove_on_error(bsg_on_error on_error);
 char *bugsnag_event_get_context(void *event_ptr);
 void bugsnag_event_set_context(void *event_ptr, char *value);
 
+
+/* Accessors for event.app */
+
+
+char *bugsnag_app_get_binary_arch(void *event_ptr);
+void bugsnag_app_set_binary_arch(void *event_ptr, char *value);
+
+char *bugsnag_app_get_build_uuid(void *event_ptr);
+void bugsnag_app_set_build_uuid(void *event_ptr, char *value);
+
+char *bugsnag_app_get_id(void *event_ptr);
+void bugsnag_app_set_id(void *event_ptr, char *value);
+
+char *bugsnag_app_get_release_stage(void *event_ptr);
+void bugsnag_app_set_release_stage(void *event_ptr, char *value);
+
+char *bugsnag_app_get_type(void *event_ptr);
+void bugsnag_app_set_type(void *event_ptr, char *value);
+
+char *bugsnag_app_get_version(void *event_ptr);
+void bugsnag_app_set_version(void *event_ptr, char *value);
+
+int bugsnag_app_get_version_code(void *event_ptr);
+void bugsnag_app_set_version_code(void *event_ptr, int value);
+
+time_t bugsnag_app_get_duration(void *event_ptr);
+void bugsnag_app_set_duration(void *event_ptr, time_t value);
+
+time_t bugsnag_app_get_duration_in_foreground(void *event_ptr);
+void bugsnag_app_set_duration_in_foreground(void *event_ptr, time_t value);
+
+bool bugsnag_app_get_in_foreground(void *event_ptr);
+void bugsnag_app_set_in_foreground(void *event_ptr, bool value);
+
```

Comment:
Should these be in declared `event.h`?

Reply:
Moved these declarations to `assets/include/event.h`

Reply:
Should we therefore take them out of `jni/event.h` and use the public header internally where we use these methods? Otherwise we have to maintain both definitions.